### DISCUSSION TITLE
终于养到猫了，第一次有养猫攻略吗

### DISCUSSION THREAD

**Discussion by LT-UT** [share121/inter-knot#3003] [其他]
![20240819110943](https://github.com/user-attachments/assets/004180a8-185b-42e9-b6f9-0d035b0d86a2)
猫名字叫NIKO，原本是流浪猫来的，六分街还有很多流浪猫，希望有人来领养

**Comment by RaphaelHider** [comment]
？你不是RandomPlay录像店的店主么，怎么出现在了绳网？你是绳匠还是盗洞客？信息安全意识这么差，一看就是刚入这行。

**Reply by LT-UT** [reply]
都不是，绳网又不是只有神匠和盗洞客才能访问，只需提供个人信息即可发帖

**Reply by RaphaelHider** [reply]
> 都不是，绳网又不是只有神匠和盗洞客才能访问，只需提供个人信息即可发帖

录像店今天应该开业吧，你等我，我要和你面基嘿嘿

**Reply by LT-UT** [reply]
？？？
不行，如果你来，我就叫治安官了

**Comment by JobNumber-9527** [comment]
你这只猫，沙鹰应该玩的很好吧<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by LT-UT** [reply]
不是那个含义，在猫名前加one shot搜一下就是她

**Comment by lllliufeng** [comment]
泻药，刚被六分街的猫抓伤了，555

**Reply by LT-UT** [reply]
请您马上来到6分街的pandom play,我就在里面任职，让我来帮你消毒，
办张vip卡呗

**Comment by nninng** [comment]
先带去绝育吧

**Reply by LT-UT** [reply]
等它什么时候发情了就去

**Comment by xiaosheng31** [comment]
建议先去做个检查然后去接种一下疫苗最后带回家时避免是否有应激反应

**Reply by LT-UT** [reply]
做了，没问题
证也办好了

**Comment by yzysrewrwqr** [comment]
竟然不叫罗小黑

**Reply by LT-UT** [reply]
这个的话，要看个人
